front/lib/actions/mcp_internal_actions/servers/process/index.ts

```diff
@@ -98,7 +98,7 @@ function makeExtractInformationFromDocumentsTool(
       dataSources: DataSourcesToolConfigurationType[number][];
       objective: string;
       jsonSchema: JSONSchema;
-      timeFrame: TimeFrame | null;
+      timeFrame?: TimeFrame;
```

Comment:
nit: technically it's always passed but can be `undefined`
```suggestion
      timeFrame: TimeFrame | undefined;
```